### DISCUSSION TITLE
How can I define in which folder the gifs created by gganimate are saved?

### DISCUSSION THREAD

**Discussion by KarenHus** [quarto-dev/quarto-cli#10132] [Q&A]
### Description

Hello, 

I am creating a HTML document with Quarto 1.3.433. Usually when rendering the document, the rendered files (png) are saved in a folder on the folder level of the qmd-document. 

Example: The folder is named _groundwater_. In _groundwater_ the file _groundwater_level.qmd_ exists. The rendered files are then created in _groundwater/groundwater_level_files/figure-html/unnamed-chunk-2-1.png_

However, when I use gganimate and the rendered file is a gif, the files are saved in the project folder (execute-dir) instead. So the gif cannot be displayed in the rendered HTML document. 

Here is how I use gganimate:

````qmd
---
title: "Animation Test"
date: today
---

```{r}
anim <- ggplot(mtcars, aes(mpg, disp)) +
  geom_point(aes(color = gear)) +
  transition_states(gear, transition_length = 2, state_length = 1) +
  enter_fade() +
  exit_fade()

animate(anim)
```
````
Any ideas why the render directory changes depending on the file type?

Thank you!

**Comment by mcanouil** [comment]
Look at gganimate documentation (*e.g.*, https://gganimate.com/reference/animate.html) or the engine documentation (`fig-path` option from knitr in your case).

Quarto is not controlling this.

**Comment by cscheid** [comment]
> I am creating a HTML document with Quarto 1.3.433

Before anything else: that version is a year old [to the day](https://github.com/quarto-dev/quarto-cli/releases/v1.3.433). Your question doesn't look directly related to Quarto, but you're missing a year's full of bugfixes, and some of them might be relevant. Thanks!
